draft-jones-perc-dtls-tunnel/draft-jones-perc-dtls-tunnel.md

```diff
@@ -317,7 +326,8 @@ than one tunnel has been established between it and a key distributor.
 prefer to allow the media distributor to send messages over more
 than one tunnel to more than one key distributor?  The latter would
 provide for higher availability, but at the cost of key distributor
-complexity.
+complexity. The former would allow the usage of a load distributor in
+front of the key distributor.
```

Comment:
Are you suggesting we demultiplex tunnel message and direct all traffic related to Association_ID 1 to Box A and Association_ID 2 to Box B, for example?  It might be good to call that out more explicitly, if so.